docs/concepts/projects.md

```diff
@@ -603,14 +611,47 @@ documentation.
 To distribute your project to others (e.g., to upload it to an index like PyPI), you'll need to
 build it into a distributable format.
 
+In Python building packages is split between the build frontend and the build backend. The build
+backend is the `[build-system]` definition in the `pyproject.toml`, e.g. `build-backend = "uv"` or
+`build-backend = "setuptools.build_meta"`. The build frontend is the command you execute for
+building source distributions and wheels, such as `uv build` or `python -m build`. You can mix
+tools: `uv build` can build a package with a setuptools backend, just like `python -m build` can use
+a uv backend.
+
 Python projects are typically distributed as both source distributions (sdists) and binary
-distributions (wheels). The former is typically a `.tar.gz` or `.zip` file containing the project's
-source code along with some additional metadata, while the latter is a `.whl` file containing
-pre-built artifacts that can be installed directly.
+distributions (wheels). The former is typically a `.tar.gz` file containing the project's source
+code along with some additional metadata, while the latter is a `.whl` file containing pre-built
+artifacts that can be installed directly. Wheel come in two flavors: Universal and platform
+specific. A universal wheel is called `{name}-{version}-py3-none-any.whl` and only needs a
+sufficiently recent Python version. Package with native code produce platform-specific wheels, e.g.,
+`numpy-2.1.2-cp312-cp312-win_amd64.whl` only works on CPython 3.12 on 64-bit x86 Windows. With
+universal wheels, the source distribution only exists for redistributors such as linux
+distributions, while with platform specific wheels, it is a fallback for platforms with no matching
+built wheel. The uv build backend currently only supports universal, pure Python projects, while the
+`uv build` frontend can build native code through specialized build backends.
+
+### Build backend
+
+The uv build backend supports pure python projects. Their code needs to be in `src/{project_name}`
+where all dots (`.`) and dashes (`-`) in the project name are replaced with underscores, e.g., the
+project `foo-bar` needs at least `src/foo_bar/__init__.py`.
+
+To use uv as build backend, add the following to `pyproject.toml`:
+
+```toml
+[build-system]
+requires = ["uv>=0.4.18,<5"]
```

Comment:
```suggestion
requires = ["uv>=0.4.18,<0.5"]
```

also, is a minor version bumb be breaking change? maybe it can `requires = ["uv>=0.4.18,<1"]`